<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>16</horstretch>
    <verstretch>9</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>16</width>
    <height>9</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>16</width>
    <height>9</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Doctor's shedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="shedule">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>1021</width>
      <height>731</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>630</y>
      <width>312</width>
      <height>190</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::NoSelection</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::ShortDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="doctors_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>121</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="surnames_table">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>281</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="add_doctor_menu"/>
    <addaction name="add_patient_menu"/>
    <addaction name="remove_doctor_menu"/>
    <addaction name="remove_patient_menu"/>
    <addaction name="edit_doc_menu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="add_doctor_menu">
   <property name="text">
    <string>Добавить врача</string>
   </property>
  </action>
  <action name="add_patient_menu">
   <property name="text">
    <string>Добавить запись</string>
   </property>
  </action>
  <action name="remove_doctor_menu">
   <property name="text">
    <string>Удалить врача</string>
   </property>
  </action>
  <action name="remove_patient_menu">
   <property name="text">
    <string>Удалить запись</string>
   </property>
  </action>
  <action name="edit_doc_menu">
   <property name="text">
    <string>Редактировать врача</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
